pkg/service/connections.go

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package service
+
+import (
+	"errors"
+	"net"
+	"syscall"
+
+	"github.com/vishvananda/netlink"
+	"golang.org/x/sys/unix"
+
+	"github.com/cilium/cilium/pkg/datapath/sockets"
+	lb "github.com/cilium/cilium/pkg/loadbalancer"
+)
+
+var opSupported = true
+
+func (s *Service) destroyConnectionsToBackend(be *lb.Backend) {
+	if !opSupported {
+		return
+	}
+	log.Infof("debug-aditi destroying connections to backend %v", be.L3n4Addr)
```

Comment:
Should we strip `debug-aditi`?

Reply:
Looks like these slipped through during rebase. I'll update them.